src/assets/scss/layout/layout.scss

```diff
@@ -0,0 +1,50 @@
+@import "../../../assets/scss/helpers/variables";
+
+.pf-c-dropdown {
+  &__menu {
+    &-item {
+      color: #000 !important;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  position: fixed;
+  height: 0.5em;
+  bottom: 0em;
+  left: 0em;
+  right: 0em;
+  width: 100%;
+
+  .footer-text {
+    color: white;
+    text-align: center;
+  }
+
+  .footer-link {
+    display: flex;
+    align-items: center;
+
+    .footer-logo {
+      width: 2em;
+      margin-left: 0.4em;
+      margin-right: 0.4em;
+    }
+  }
+}
+
+.footer-styling {
+  margin-bottom: 0.5em;
+}
+
+.pf-c-page__header-tools:nth-child(3) {
+  justify-content: flex-end;
+}
+
+.logo {
+  margin-right: 4rem;
+  padding: 0.8rem;
+  height: 70px;
+  width: 190px;
```

Comment:
Might need to specify a `min-height` here.
![Screenshot from 2021-03-08 14-26-11](https://user-images.githubusercontent.com/24356368/110370891-44666b80-801a-11eb-8e97-207dc78e6ab0.png)